platform-sdk/docs/proposals/metric-labels/README.md

```diff
@@ -0,0 +1,166 @@
+# Metric labels
+
+author: @hendrikEbbers  
+status: draft
```

Comment:
Not sure if this makes sense, since as soon it's accepted you need to push a change and need reviews again

Reply:
No. The idea is: as long as it is a draft we should discuss without voting ;) Than it will be moved to `voting` and people can vote. We can even add `implementation in progress`, `implemented` and `declined`. From my point of view we need to store the state in the proposal. How will you otherwise know if a proposal that is merged is already implemented?

Reply:
For this we have the board I guess

Reply:
Once it is commited you want to have a state in the file